src/definitions.json

```diff
@@ -2,14 +2,15 @@
   "id": "The unique integer id of the node used internally by Hetmech (but not in Hetionet v1.0)",
   "metapath": "The type of path (metapath) connecting the source node to the target node",
   "path_count": "The number of paths between the source and target node of the specified metapath (path type)",
+  "adjusted_p_value": "Adjusted p-value",
   "p_value": "A measure of the significance of the DWPC that indicates whether more paths were observed than expected due to random chance. Compares the DWPC to a null distribution of DWPCs generated from degree-preserving permuted networks.",
```

Comment:
We can add explanation here as well:

> A measure of the significance of the DWPC that indicates whether more paths were observed than expected due to random chance. Compares the DWPC to a null distribution of DWPCs generated from degree-preserving permuted networks. **This p-value has not been adjusted for multiple comparisons (i.e. when multiple metapaths are assessed for significant connectivity between the source and target node)**.

Reply:
Will add. What about a better description for adjusted p value. "Adjusted p-value" was just a placeholder. Will need your expertise.